configs/mainnet.yaml

```diff
@@ -61,6 +61,9 @@ GLOAS_FORK_EPOCH: 18446744073709551615
 # EIP7441
 EIP7441_FORK_VERSION: 0x08000000  # temporary stub
 EIP7441_FORK_EPOCH: 18446744073709551615
+# EIP7928
+EIP7928_FORK_VERSION: 0x09000000  # temporary stub
+EIP7928_FORK_EPOCH: 18446744073709551615
 # EIP7805
 EIP7805_FORK_VERSION: 0x0a000000  # temporary stub
 EIP7805_FORK_EPOCH: 18446744073709551615
```

Comment:
These items are sorted alphabetically. Please put the eip7928 values after eip7805. Here and other places. Maybe use `0x0b000000` for the fork version instead, doesn't really matter.